packages/app/src/overview/components/device-preview/device-preview-test-ids.enum.ts

```diff
@@ -8,4 +8,5 @@ export enum DeviceTestIds {
   PureGray = "pure-gray",
   PureBlack = "pure-black",
   PureSystemButton = "pure-system-button",
+  SerialNumber = "device-serial-number",
```

Comment:
thanks for the device prefix. It's a good practice, to add prefixes to avoid conflicts in the feature with a global enum 😉 👌 

🟢 (nice to have - optional request)
If you would like to, you can modify the rest of these object values.